recipes/r-nasaaccess/meta.yaml

```
@@ -0,0 +1,104 @@
+ 
+  number: 0
+  
+  noarch: generic
+  
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+
+requirements:
+  build:
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}git
+    - {{ posix }}zip               # [win]
+
+  host:
+    - r-base =4.0.5
+    - r-xml
+    - r-ggmap
+    - r-ggplot2
+    - r-httr
+    - r-maptools
+    - r-ncdf4
+    - r-raster
+    - r-rgdal
+    - r-rgeos
+    - r-sf
+    - r-shapefiles
+    - r-sp
+    - r-stringr
+
+  run:
+    - r-base =4.0.5
+    - r-xml
+    - r-ggmap
+    - r-ggplot2
+    - r-httr
+    - r-maptools
+    - r-ncdf4
+    - r-raster
+    - r-rgdal
+    - r-rgeos
+    - r-sf
+    - r-shapefiles
+    - r-sp
+    - r-stringr
+
+test:
+  commands:
+    
+    - $R -e "library('NASAaccess')"           # [not win]
+    - "\"%R%\" -e \"library('NASAaccess')\""  # [win]
+
+about:
+  home: https://github.com/nasa/NASAaccess
+  license: NASA-1.3
+  summary: A tool to generate gridded ascii tables and station files needed by hydrological models
+    for weather and climate inputs. The package assumes that users have already set
+    up a registration account(s) with Earthdata login as well as authorizing NASA GES
+    DISC data access. Please refer to  <https://disc.gsfc.nasa.gov/data-access> for
+    further details. The package relies on 'curl' <https://curl.se/> commands and library
+    to access NASA remote sensing servers. Users should make sure that their local machines
+    have 'curl' installed properly.  Instructions on creating the ".netrc" and ".urs_cookies"
+    files can be accessed at <https://wiki.earthdata.nasa.gov/display/EL/How+To+Access+Data+With+cURL+And+Wget>.
+    Creating the ".netrc" file at the user machine 'Home' directory and storing the
+    NASA GES DISC user logging information in it is needed to execute the package commands.
+  description: |
+    A tool to generate gridded ascii tables and station files needed by hydrological models
+    for weather and climate inputs. The package assumes that users have already set
+    up a registration account(s) with Earthdata login as well as authorizing NASA GES
+    DISC data access. Please refer to  <https://disc.gsfc.nasa.gov/data-access> for
+    further details. The package relies on 'curl' <https://curl.se/> commands and library
+    to access NASA remote sensing servers. Users should make sure that their local machines
+    have 'curl' installed properly.  Instructions on creating the ".netrc" and ".urs_cookies"
+    files can be accessed at <https://wiki.earthdata.nasa.gov/display/EL/How+To+Access+Data+With+cURL+And+Wget>.
+    Creating the ".netrc" file at the user machine 'Home' directory and storing the
+    NASA GES DISC user logging information in it is needed to execute the package commands.
+  license_family: OTHER
+  license_file:
+    - LICENSE
+  doc_url: https://imohamme.github.io/NASAaccess/
+  dev_url: https://github.com/imohamme/NASAaccess
+
+
+extra:
+  recipe-maintainers:
+    - imohamme
```

Please also add the maintainer group. thanks!
```suggestion
    - conda-forge/r
    - imohamme
```

---

@dbast Thanks for looking at r-nasaaccess. The maintainer group has been added as directed and the recipe is ready for your review.